src/backend/distributed/commands/multi_copy.c

```diff
@@ -1217,6 +1228,209 @@ ReportCopyError(MultiConnection *connection, PGresult *result)
 }
 
 
+static void
+ConversionPathForTypes(Oid inputType, Oid destType, CopyCoercionData *result)
+{
+	Oid coercionFuncId = InvalidOid;
+	CoercionPathType coercionType;
+
+	if (destType == inputType)
+	{
+		result->coercionType = COERCION_PATH_RELABELTYPE;
+		return;
+	}
+
+	coercionType = find_coercion_pathway(destType, inputType,
+										 COERCION_EXPLICIT,
+										 &coercionFuncId);
+
+	switch (coercionType)
+	{
+		case COERCION_PATH_NONE:
+		{
+			ereport(ERROR, (errmsg("can not coerce %d to %d", inputType, destType)));
```

Comment:
postgres uses "cannot cast" for these messages, because coercion is not a word used by mortals.

Reply:
This isn't strictly true:

```
~/D/C/postgresql ❯❯❯ ack 'errmsg.*coerc'
src/backend/catalog/pg_aggregate.c
860:					 errmsg("function %s requires run-time type coercion",

src/backend/parser/parse_oper.c
480:				 errmsg("operator requires run-time type coercion: %s",

src/backend/commands/functioncmds.c
1490:					 errmsg("argument of cast function must match or be binary-coercible from source data type")));
1504:					 errmsg("return data type of cast function must match or be binary-coercible to target data type")));

src/backend/commands/prepare.c
380:					 errmsg("parameter $%d of type %s cannot be coerced to the expected type %s",
```

That said, _cannot cast_ is more common in `parse_coerce.c`, so ultimately agreed.